### DISCUSSION TITLE
When converting from md to docx, the br tags in tables were ignored.

### DISCUSSION THREAD

**Discussion by CBAnother** [jgm/pandoc#9536] [Q&A] [answered]
The markdown content is as follows

```markdown
|      | Column1    | Column2    |
| ---- | ---------- | ---------- |
| Row1 | aa         | bb<br />ff |
| Row2 | cc<br />dd | ee         |
```

The commands executed are as follows.

```
pandoc "test.md" -o "test.docx"
```

The result is as shown, `<br />` was ignored

![image](https://github.com/jgm/pandoc/assets/62218147/5bd1df53-9a20-48ad-b11c-81e1f16ab701)

**Comment by jgm** [comment]
Yes, as documented. Raw HTML is treated as raw HTML and passed through to formats that accept raw HTML (e.g. HTML and Markdown). It is ignored in other formats (e.g. docx).

**Reply by CBAnother** [reply]
How can I prevent it from being ignored?

**Reply by jgm** [reply, marked as answer]
You could write a Lua filter that matches a RawInline element with format "html" and contents "<br>" and replaces it with a native pandoc LineBreak element.  See the docs on lua filters.

**Reply by CBAnother** [reply]
Okay, thank you so much :), I'll go learn about Lua first and then give it a try

**Reply by CBAnother** [reply]
Thank you so much, I made it! 
```lua
function LineBreak(elem)
  -- Replace raw html line breaks with pandoc line breaks
  if elem.t == "RawInline" and elem.format == "html" and 
      (elem.text == "<br>" or elem.text == "<br />") then
      return pandoc.LineBreak()
  end
end

return {
  {RawInline = LineBreak}
}
```
